src/NUnitFramework/tests/Assertions/AssertThrowsTests.cs

```diff
@@ -101,7 +101,7 @@ public void UnrelatedExceptionThrown()
             var ex = CatchException(() => Assert.Throws<ArgumentException>(TestDelegates.ThrowsNullReferenceException));
             Assert.That(ex.Message, Does.StartWith(
                 "  Expected: <System.ArgumentException>" + Env.NewLine +
-                "  But was:  <System.NullReferenceException>" + Env.NewLine));
+                "  But was:  <System.NullReferenceException: my message" + Env.NewLine + "   at NUnit.TestUtilities"));
```

Comment:
You may run into cross-platform issues making assumptons about what's on top of the stack. We can wait and see, however.